### ISSUE TITLE
DPM Partition Dump is broken due to "operation_timeout" parameter

### ISSUE THREAD

**Issue by eaibmz** [zhmcclient/zhmccli#250]
Trying to trigger a DPM partition dump.

### Actual behavior

```
2021/11/25 03:16:49 TRACE: zhmc -h <host> -o json partition dump --volume <volume> <cpc> <partition>

2021/11/25 03:16:55 DEBUG: Traceback (most recent call last):
2021/11/25 03:16:55 DEBUG:   File "/linux390/zhmccli/bin/zhmc", line 8, in <module>
2021/11/25 03:16:55 DEBUG:     sys.exit(cli())
2021/11/25 03:16:55 DEBUG:   File "/linux390/zhmccli/lib/python3.9/site-packages/click/core.py", line 829, in __call__
2021/11/25 03:16:55 DEBUG:     return self.main(*args, **kwargs)
2021/11/25 03:16:55 DEBUG:   File "/linux390/zhmccli/lib/python3.9/site-packages/click/core.py", line 782, in main
2021/11/25 03:16:55 DEBUG:     rv = self.invoke(ctx)
2021/11/25 03:16:55 DEBUG:   File "/linux390/zhmccli/lib/python3.9/site-packages/click/core.py", line 1259, in invoke
2021/11/25 03:16:55 DEBUG:     return _process_result(sub_ctx.command.invoke(sub_ctx))
2021/11/25 03:16:55 DEBUG:   File "/linux390/zhmccli/lib/python3.9/site-packages/click/core.py", line 1259, in invoke
2021/11/25 03:16:55 DEBUG:     return _process_result(sub_ctx.command.invoke(sub_ctx))
2021/11/25 03:16:55 DEBUG:   File "/linux390/zhmccli/lib/python3.9/site-packages/click/core.py", line 1066, in invoke
2021/11/25 03:16:55 DEBUG:     return ctx.invoke(self.callback, **ctx.params)
2021/11/25 03:16:55 DEBUG:   File "/linux390/zhmccli/lib/python3.9/site-packages/click/core.py", line 610, in invoke
2021/11/25 03:16:55 DEBUG:     return callback(*args, **kwargs)
2021/11/25 03:16:55 DEBUG:   File "/linux390/zhmccli/lib/python3.9/site-packages/click/decorators.py", line 33, in new_func
2021/11/25 03:16:55 DEBUG:     return f(get_current_context().obj, *args, **kwargs)
2021/11/25 03:16:55 DEBUG:   File "/linux390/zhmccli/lib/python3.9/site-packages/zhmccli/_cmd_partition.py", line 238, in partition_dump
2021/11/25 03:16:55 DEBUG:     cmd_ctx.execute_cmd(lambda: cmd_partition_dump(
2021/11/25 03:16:55 DEBUG:   File "/linux390/zhmccli/lib/python3.9/site-packages/zhmccli/_helper.py", line 327, in execute_cmd
2021/11/25 03:16:55 DEBUG:     cmd()
2021/11/25 03:16:55 DEBUG:   File "/linux390/zhmccli/lib/python3.9/site-packages/zhmccli/_cmd_partition.py", line 238, in <lambda>
2021/11/25 03:16:55 DEBUG:     cmd_ctx.execute_cmd(lambda: cmd_partition_dump(
2021/11/25 03:16:55 DEBUG:   File "/linux390/zhmccli/lib/python3.9/site-packages/zhmccli/_cmd_partition.py", line 1325, in cmd_partition_dump
2021/11/25 03:16:55 DEBUG:     partition_dump_with_sm(cmd_ctx, partition, **options)
2021/11/25 03:16:55 DEBUG:   File "/linux390/zhmccli/lib/python3.9/site-packages/zhmccli/_cmd_partition.py", line 1365, in partition_dump_with_sm
2021/11/25 03:16:55 DEBUG:     operation_timeout=options['operation_timeout'])
2021/11/25 03:16:55 DEBUG: KeyError: 'operation_timeout'
```

It seems that the problem is due to `options = original_options(options)` which converts "operations_timeout" to "operations-timeout"
https://github.com/zhmcclient/zhmccli/blob/53e89db484795653e10f98aa33be5e1f42a51b37/zhmccli/_cmd_partition.py#L1322

But the partition code uses this: `operation_timeout=options['operation_timeout']`

### Execution environment

* zhmccli version: latest master branch
* zhmcclient version: latest master branch
* Operating system (type+version): Ubuntu 21.10
* HMC version: 2.15.0
* CPC version: ???

**Comment by eaibmz** [comment]
Furthermore, after fixing the options bug manually i'm hitting this issue whenever i user -T:

```
Error: HTTPError: 400,6: The JSON object contains an unrecognized field ('operation_timeout') [POST /api/partitions/..../operations/start-dump-program]
```

**Comment by eaibmz** [comment]
My dirty options fix:
```
@@ -1319,7 +1319,7 @@

     client = zhmcclient.Client(cmd_ctx.session)
     partition = find_partition(cmd_ctx, client, cpc_name, partition_name)
-    options = original_options(options)
+    #options = original_options(options)

     if storage_management_feature(partition):
         partition_dump_with_sm(cmd_ctx, partition, **options)
```

**Comment by andy-maier** [comment]
The options come in to the zhmccli code (from Click) with underscores in their names. The HMC requires dashes. That is why the original_options() function is needed, so your quick fix does not work as you saw.

The bug needs to be fixed in the zhmccli code which should either use the original options with underscore, or the HMC options with dash.

**Comment by andy-maier** [comment]
The fix has been rolled back to zhmccli 1.0.3, to be released shortly.
